build.gradle

```diff
@@ -55,6 +55,9 @@ buildConfig {
     buildConfigField 'String', 'SENTRY_DSN', 'https://0263d6473bd24a9ba40e25aa5fb0a242:c5451dc815074bff8ce3fb9f0851f2f5@sentry.io/233260'
     buildConfigField 'boolean', 'PRINT_STACK_TRACE', 'true'
 
+    //Models storage path
+    buildConfigField 'String', 'LIBRARY_MODELS_URL', 'https://storage.googleapis.com/lib_models/pkl/'
```

Comment:
Account version of models, e.g. `'https://storage.googleapis.com/sourcerer-models/v1/'`.
Also, we should use single google cloud account for servers and models.

Reply:
sourcerer-app/ml_models/v1 ?

Reply:
Moved.